Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3-alpine
+
+MAINTAINER "CloudOps ClarkSource <CloudOps@clark.de>"
+LABEL maintainer="CloudOps@clark.de"
+
+RUN apk add --upgrade apk-tools --no-cache && apk upgrade --available --no-cache && apk add git py3-setuptools --no-cache
```

Comment:
do you really need to add apk-tools, py3-setuptools, and upgrading the system? 

If so, you could definitely merge both runs and clean tools that were only used for setup.

196MB vs 137MB using something similar to this:
```
FROM python:3-alpine

MAINTAINER "CloudOps ClarkSource <CloudOps@clark.de>"
LABEL maintainer="CloudOps@clark.de"

COPY . /setup
WORKDIR /setup

RUN \
  apk add --no-cache --upgrade git && \
  python3 setup.py install && \
  apk del git && \
  rm -rf /setup /var/cache/apk

EXPOSE 8000/tcp
CMD ["aws-exporter"]
```

Reply:
no preferences, tbh I have no idea why we even need git there, it's just a python scripts and we can just copy it manually to appropriate directory.